src/components/AppNavigation/Settings/SettingsAddressbook.vue

```diff
@@ -24,11 +24,19 @@
 	<div class="settings-addressbook-list">
 		<IconContact class="settings-line__icon" />
 		<li :class="{'addressbook--disabled': !addressbook.enabled}" class="addressbook">
-			<!-- addressbook name -->
-			<span class="addressbook__name" :title="addressbook.displayName">
-				{{ addressbook.displayName }}
-			</span>
+			<div class="addressbook__content">
+				<!-- addressbook name -->
+				<span class="addressbook__name" :title="addressbook.displayName">
+					{{ addressbook.displayName }}
+				</span>
 
+				<!-- counters -->
+				<div class="addressbook__count-wrapper">
+					<span class="addressbook__count">{{ groupsCount }} contact(s)</span>
+					<span class="addressbook__count">- {{ contactsCount }} group(s)</span>
```

Comment:
```suggestion
					<span class="addressbook__count">{{ n('contacts', '%n contact', '%n contacts', contactsCount) }} contact(s)</span>
					<span class="addressbook__count">- {{ n('contacts', '%n group', '%n groups', contactsCount) }} group(s)</span>
```

First parameter is the app name
Second parameter is the singular translation
Third parameter is the plural translation
Fourth parameter is the number of elements

https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html?highlight=translation#javascript

groupsCount and contactsCount were mixed up